index.html

```diff
@@ -4,484 +4,528 @@
+                  src="./src/icons/search-icon.svg"
+                  alt="search"
                 />
-              </div>
 
-              <div class="right-wrapper__flying-icon paypal-icon">
-                <img
-                  class="paypal-img"
-                  src="./src/icons/logos_paypal.svg"
-                  alt="paypal icon"
+                <input
+                  class="search-input"
+                  type="text"
+                  placeholder="Job title or Keyword"
+                  name="text"
                 />
               </div>
 
-              <div class="right-wrapper__flying-icon indeed-icon">
+              <span class="line"></span>
+
+              <div class="search-area">
                 <img
-                  class="indeed-img"
-                  src="./src/icons/simple-icons_indeed.svg"
-                  alt="indeed icon"
+                  class="location-icon"
+                  src="./src/icons/location-icon.svg"
+                  alt="location"
                 />
-              </div>
 
-              <div class="right-wrapper__flying-icon slack-icon">
-                <img
-                  class="slack-img"
-                  src="./src/icons/grommet-icons_slack.svg"
-                  alt="slack icon"
+                <input
+                  class="location-input"
+                  type="text"
+                  placeholder="Location"
+                  name="text"
+                  value=""
                 />
               </div>
 
-              <!-- First flying card -->
-              <div class="right-wrapper__flying-card first-flying-card">
-                <div class="flying-card__img-block first-img">
-                  <div class="flying-card__circle">
-                    <img src="./src/icons/star.svg" alt="star" />
-                    <p class="flying-card__rating">4.8</p>
-                  </div>
-                </div>
-                <h2 class="flying-card__title">Maliha</h2>
-                <p class="flying-card__subtitle">UX Designer</p>
+              <button type="submit" class="button search-button" href="#">
+                Search
+              </button>
+            </form>
+          </div>
+        </div>
+
+        <!-- Graphics -->
+        <div class="graphics">
+          <img
+            class="graphics__img"
+            src="./src/img/satisfied-lovely-woman-holds-modern-cell-phone-1.png"
+            alt="A satisfied lovely woman holds a modern cell phone"
+          />
+
+          <!-- Big ellipse -->
+          <img
+            class="graphics__ellipse-big"
+            src="./src/img/ellipse-big.svg"
+            alt="big ellipse"
+          />
+          <!-- Small ellipse -->
+          <img
+            class="graphics__ellipse-small"
+            src="./src/img/ellipse-small.svg"
+            alt="small ellipse"
+          />
+
+          <!-- Flying icons -->
+          <div class="graphics__flying-icon google-icon">
```

Comment:
```suggestion
          <div class="graphics__floating-icon">
```
1. It's not critical, but I think 'floating' is a more suitable word.
2. You may write in CSS a selector `.graphics__floating-icon .google-img {}` and rid off extra classes for every icon.
3. Please, apply this comment to other icons below too.

Reply:
okay, but i need classes like google-icon, because google-img inside google-icon and i need to transform img AND icon